src/test_workflow/integ_test/service_opensearch.py

```diff
@@ -33,7 +35,9 @@ def __init__(
         security_enabled: bool,
         dependency_installer: DependencyInstaller,
         work_dir: str,
-        cluster_port: int = 9200
+        cluster_port: int = 9200,
+        trans_stdout: str = "",
+        trans_stderr: str = ""
```

Comment:
Why is the value initialize to empty?

Reply:
sorry, It was left over from my other earlier attempts,, I have them removed from my latest update. Thank you for noticing that..